webui_runner/plugs/wasmtime_app/echo_repo.sh

```diff
@@ -0,0 +1 @@
+echo https://github.com/CraneStation/wasmtime
```

Comment:
If things get renamed, this could change; perhaps this should also be determined dynamically?